modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -394,11 +394,21 @@ public long getUserId() {
 	}
 
 	public boolean isSwapConfigurationEnabled() {
-		if (_assetVocabularies.size() == 2) {
-			return true;
+		if (_swapConfigurationEnabled != null) {
```

Comment:
Why not return swapConfigurationEnabled directly?

Reply:
Is this a mutable value? Could it not being initialized in the constructor?

Reply:
that's true, I'm following the same approach as with the rest of properties. But I really like this way you are proposing

Reply:
but i'm going to keep it as it is, other way I may feel uncomfortable having some  properties being initialized in the constructor and others in its getter method. But this is something we can consider. I agree with you in setting variable as final and initializing them in the constructor.